src/content/docs/data-apis/manage-data/manage-data-coming-new-relic.mdx

```diff
@@ -19,6 +19,12 @@ When you connect your data to New Relic, we process what we receive and apply da
   This doc is for accounts on our [New Relic One pricing model](/docs/accounts/accounts-billing/new-relic-one-pricing-users/pricing-billing). If you're on our original product-based pricing model, see [Original data retention](/docs/accounts/original-accounts-billing/product-based-pricing/overview-data-retention-components). Not sure which you're on? See [Overview of pricing and user model](/docs/transition-guide-our-new-pricing-plan-user-model).
 </Callout>
 
+To learn how to easily analyze the data your account is ingesting, watch this short video (3:18 minutes).
+
+<Video id="3mDiExsIjhE" type="youtube" />
+
```

Comment:
SO many videos!

Reply:
understatement :-) And there are still approx 30 more!